test/compare.js

```diff
@@ -0,0 +1,96 @@
+// Copyright © 2023 IBM Corp. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const _ = require('lodash');
```

Comment:
I wonder if it might be nicer to just import the specific functions we need
```suggestion
const chunk = require("lodash/chunk");
const difference = require("lodash/difference");
const forOwn = require("lodash/forOwn");
const isEmpty = require("lodash/isEmpty");
const union = require("lodash/union");
```